react/features/conference/components/native/carmode/microphone-button/styles.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import BaseTheme from '../../../../../base/ui/components/BaseTheme.native';
+
+const DEFAULT_SIZE = 65;
+
+/**
+ * The styles of the microphone button.
+ */
+export default {
+
+    container: (size: number = DEFAULT_SIZE) => {
```

Comment:
Just a pure curiosity. 
Why don't we set DEFAULT_SIZE to 180 from the start? I see that we only use MicrophoneButton inside Conference.